src/synopsys-action/synopsys-bridge.ts

```diff
@@ -290,4 +311,14 @@ export class SynopsysBridge {
     }
     return false
   }
+
+  async setBridgeExecutablePath(osName: string, filePath: string): Promise<string> {
+    if (osName === 'win32') {
+      this.bridgeExecutablePath = await tryGetExecutablePath(filePath.concat('\\synopsys-bridge'), ['.exe'])
```

Comment:
I believe for execution, we are already setting the executable path at the time of executing. Please test it thoroughly for all the cases, if setting here does not create any issue.